okhttp-backend/monix/src/test/scala/sttp/client3/okhttp/monix/OkHttpMonixHttpTest.scala

```diff
@@ -18,6 +18,8 @@ class OkHttpMonixHttpTest extends HttpTest[Task] {
 
   override def supportsDeflateWrapperChecking = false
 
+  override def supportsNonAsciiHeaderValues = false
```

Comment:
This is where I'd like to use `OkHttpHttpTest` base class but I can't because there's no `test->test` dependency between modules.

Reply:
ah ... sure, here's that fine - and we already have this kind of dep in the async http client tests for example, no?